doc/source/reference/ufuncs.rst

```diff
@@ -429,9 +429,9 @@ Attempting to call these methods on other ufuncs will cause a
 :exc:`ValueError`. The reduce-like methods all take an *axis* keyword
 and a *dtype* keyword, and the arrays must all have dimension >= 1.
 The *axis* keyword specifies the axis of the array over which the reduction
-will take place and may be negative, but must be an integer. The
+will take place and may be negative, in which case it counts from the last to the first axis. *axis* can be an integer, or if in :meth:`ufunc.reduce`, it can also be `None` or `tuple of ints` . The
```

Comment:
I'm sorry, you're right that this should only be for `reduce`. How about
```
Generally, it is an integer, though for `:meth:`ufunc.reduce`, it can also be a
a tuple of `int` to reduce over several axes at once, or `None`, to reduce over
all axes.
```

Small detail: please stick to <80 characters per line.